scripts/fiber/tests-failing.txt

```diff
@@ -1,6 +1,27 @@
 src/isomorphic/classic/__tests__/ReactContextValidator-test.js
 * should pass previous context to lifecycles
 
+src/renderers/__tests__/ReactComponentLifeCycle-test.js
+* should carry through each of the phases of setup
+
+src/renderers/__tests__/ReactComponentTreeHook-test.js
+* can be retrieved by ID
+
+src/renderers/__tests__/ReactHostOperationHistoryHook-test.js
+* gets recorded during an update
+
+src/renderers/__tests__/__tests__/ReactPerf-test.js
```

Comment:
I'll move these up to `src/renderers/__tests__/` instead of the unnecessary extra level.